### DISCUSSION TITLE
Lesson 7 : Error "Call the contract method without broadcasting a transaction."

### DISCUSSION THREAD

**Discussion by irwingtello** [smartcontractkit/full-blockchain-solidity-course-py#306] [Q&A] [answered]
Can you help me with this error?
https://github.com/smartcontractkit/full-blockchain-solidity-course-py/issues/302

**Comment by cromewar** [comment]
Hi irwing I wonder why are you importing the contracts from node_modules instead of @chainlink? 

regarding to the Virtual machine error it seams is has to be with your `getEntranceFee` function, could you please paste your deploy and test  scripts here?

**Reply by irwingtello** [reply]
Sure, this is my repo:
https://github.com/irwingtello/lottery

And my error:
Compiling contracts... Solc version: 0.8.9 Optimizer: Enabled Runs: 200 EVM Version: Istanbul CompilerError: solc returned the following errors:

ParserError: Source "C:/Users/irwin/.brownie/packages/smartcontractkit/chainlink-brownie-contracts@1.1.1/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol" not found: File not found. --> contracts/Lottery.sol:4:1: | 4 | import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol"; | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

ERROR: Unable to load project

**Comment by cromewar** [comment, marked as answer]
Hello Irwing the error is the following your are importing the  `AggregatorV3Interface` in version 0.8 and on your `brownie-config.yaml`you are using the dependency on version 1.1.1:

```YAML
 dependencies:
  - smartcontractkit/chainlink-brownie-contracts@1.1.1
compiler:
  solc:
    remappings:
      - '@chainlink=smartcontractkit/chainlink-brownie-contracts@1.1.1'
```

If you check the github of chainlink-brownie-contracts version 1.1.1 you will notice there is not a version 0.8:

https://github.com/smartcontractkit/chainlink-brownie-contracts/tree/1.1.1

So the solution is, to change the version you are importing from 1.1.1 to 0.2.2 and this should work.

**Reply by irwingtello** [reply]
Thanks :D!
